specification/metrics/billedcost.md

```diff
@@ -0,0 +1,30 @@
+# Billed Cost
+
+Billed Cost represents a charge inclusive of negotiated discounts that a consumer would be charged for each billing period. Billed Cost does not amortize up-front charges (one-time or recurring). Billed Cost is often used to perform FinOps capabilities that require cash-basis accounting such as cost allocation, budgeting, and invoice reconciliation.
```

Comment:
Don't organizations use amortized cost for cost allocation?

Reply:
Only if a central team is paying the up-front purchase cost. We used to, and charge amortized cost + cost of funds but that's not always the case.

Reply:
Isn't upfront 1 word?

Reply:
Yes, 1 word.